test/integration/ignored.toml

```diff
@@ -21,18 +26,74 @@ definitions = '''
     """
     config_suffix = """
     [[codegen.ignore]]
-    type = "Foo"
-    members = ["a"]
+    type = "ns_ignored::Bar"
 
+    """
+    expect_json = '''[{
+      "staticSize":96,
+      "exclusiveSize":96,
+      "members":[]
+      }]'''
+
+  [cases.subtype]
+    skip = 'v2 hides the member entirely when it should show it with static size' # todo: github issue
```

Comment:
```suggestion
```

no need to skip these anymore now v1 is deprecated, if the behaviour changes in the future then the tests should be updated accordingly